npm/examples/windows/azure-npm.yaml

```diff
@@ -108,8 +107,6 @@ spec:
                   fieldPath: spec.nodeName
             - name: NPM_CONFIG
               value: .\\etc\\azure-npm\\azure-npm.json
-            - name: PATH
```

Comment:
remove our first attempt at mitigation, which is not back-compat and is incomplete (can't use vfpctrl and other tools used by HNS to collect logs)